app/sfr-search-api/lib/v3Edition.js

```diff
@@ -145,6 +145,7 @@ class V3Edition {
    */
   sortInstances() {
     // Sort instances by holdings
+    // TODO sort() has changed in nodejs 11>, update this function to reflect that
```

Comment:
oh no

Reply:
yeah, tried going to 12 locally and bad things happened. We can stay on 10 for a while, but need to figure out how to handle this.

(They actually did make some improvements to it, it's just a pain that it changes the order in what this returns)